indigo_app/js/components/toc-controller/index.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <div class="toc">
+    <slot name="search"
```

Comment:
I think it would be much easier if these weren't included in this TOC component at all. Instead, move them up into the wrapping component and simply call the necessary methods / set the necessary data attributes on this component.

That would remove the need for defaults, and fix the data flow (up/down) problem.

Reply:
done